etc/testing/circle/workloads/go-test-results/egress/main.go

```diff
@@ -0,0 +1,194 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jackc/pgconn"
+	_ "github.com/jackc/pgx/v4/stdlib"
```

Comment:
We can start using /v5/ for new code, I think.  We should migrate our /v4 code to it eventually, but no need to start outdated on day 1.

Reply:
I tried this and I have to chose between v5 and using dbutil :/ 
I went with using the dbutil.GetDB and hopefully when that upgrades, this file should upgrade with an import change.

Reply:
that's great.  i'm glad we're using the same libraries/helper functions for everything.